packages/universal-xml-plugin/lib/xpath.js

```diff
@@ -4,6 +4,7 @@ import _ from 'lodash';
 
 export function runQuery(query, xmlStr) {
   const dom = new DOMParser().parseFromString(xmlStr, MIME_TYPE.XML_TEXT);
+  // @ts-expect-error Misssing Node properties are not needed.
```

Comment:
maybe add a link to the actual issue here, so we might remove the rule after it's fixed